docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -1,47 +1,70 @@
 [[synthetics-create-test]]
 = Write a synthetic test
 
-++++
-<titleabbrev>Write a synthetic test</titleabbrev>
-++++
-
 [[synthetics-syntax]]
 
 beta[] To write synthetic tests for your application, you'll need to know basic JavaScript and
 {playwright-url}[Playwright] syntax.
-The synthetics agent exposes an API for creating and running tests, including:
 
-[horizontal]
-`journey`::     Tests one discrete unit of functionality.
-                Takes two parameters: a `name` (string) and a `callback` (function).
-                See <<synthetics-create-journey>>.
-`step`::        Actions within a journey that should be completed in a specific order.
-                Takes two parameters: a `name` (string) and a `callback` (function).
-                See <<synthetics-create-step>>.
-`beforeAll`::   Runs a provided function once, before any `journey` runs.
-                If the provided function is a promise, the runner will wait for the
-                promise to resolve before invoking the `journey`.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`before`::      Runs a provided function before a single `journey` runs.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`afterAll`::    Runs a provided function once, after all the `journey` runs have completed.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`after`::       Runs a provided function after a single `journey` has completed.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
+// The synthetics agent exposes an API for creating and running tests, including:
```

Comment:
I don't like starting this guide with reference-like material, but I'm not sure where else to put this unless we have a reference page similar to the [command reference](https://observability-docs_1925.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-command-reference.html).